src/scripts/main.js

```diff
@@ -21,7 +20,19 @@
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (let key in data) {
+    const li = document.createElement('li');
+    li.textContent = key;
+    ul.appendChild(li);
+
+    if (typeof data[key] === 'object' && Object.keys(data[key]).length > 0) {
+      li.appendChild(createTree(li, data[key]));
+    }
+  }
+
+  return ul;
 }
 
-createTree(tree, food);
+tree.appendChild(createTree(tree, food));
```

Comment:
You are appending the created tree to the `tree` element. This operation can cause a reflow, which is expensive in terms of performance. To avoid unnecessary reflows, you could create and append all list items to a document fragment before appending the fragment to the `tree` element.